bisqapps/shared/presentation/build.gradle.kts

```diff
@@ -34,25 +38,22 @@ buildConfig {
         buildConfigField("SHARED_LIBS_VERSION", project.version.toString())
         buildConfigField("BUILD_TS", System.currentTimeMillis())
     }
-//    buildConfigField("APP_SECRET", "Z3JhZGxlLWphdmEtYnVpbGRjb25maWctcGx1Z2lu")
-//    buildConfigField<String>("OPTIONAL", null)
-//    buildConfigField("FEATURE_ENABLED", true)
-//    buildConfigField("MAGIC_NUMBERS", intArrayOf(1, 2, 3, 4))
-//    buildConfigField("STRING_LIST", arrayOf("a", "b", "c"))
-//    buildConfigField("MAP", mapOf("a" to 1, "b" to 2))
-//    buildConfigField("FILE", File("aFile"))
-//    buildConfigField("URI", uri("https://example.io"))
-//    buildConfigField("com.github.gmazzo.buildconfig.demos.kts.SomeData", "DATA", "SomeData(\"a\", 1)")
+    //    buildConfigField("APP_SECRET", "Z3JhZGxlLWphdmEtYnVpbGRjb25maWctcGx1Z2lu")
+    //    buildConfigField<String>("OPTIONAL", null)
+    //    buildConfigField("FEATURE_ENABLED", true)
+    //    buildConfigField("MAGIC_NUMBERS", intArrayOf(1, 2, 3, 4))
+    //    buildConfigField("STRING_LIST", arrayOf("a", "b", "c"))
+    //    buildConfigField("MAP", mapOf("a" to 1, "b" to 2))
+    //    buildConfigField("FILE", File("aFile"))
+    //    buildConfigField("URI", uri("https://example.io"))
+    //    buildConfigField("com.github.gmazzo.buildconfig.demos.kts.SomeData", "DATA",
+    // "SomeData(\"a\", 1)")
```

Comment:
there are some spacing / indentation changes in your commits, can you review it? we use the default fleet IDE setup

Reply:
I'm also using fleet, don't know why this changes came up

Reply:
I know why nvim